tools/generator/src/main.rs

```diff
@@ -0,0 +1,172 @@
+//! CLI for generating iroha sample configuration and genesis, as well
```

Comment:
can we name this tool more appropriately? like `configuration_generator`?

Reply:
if it's some sort of general purpose generator can we place `crypto_cli` under it. It is also a form of a generator

Reply:
if docs are always related to sample configuration I would call it something like `confgen` or `samplegen` etc.

Reply:
iroha_gen seems generic-enough for me.

Reply:
After some deliberation we agreed that the best name for this tool would be `kagami`.

Reply:
We could name it `iroha_gen` for now and rename it when this tool also has a teacher-like role